src/tracing_tool_output.ml

```diff
@@ -15,17 +15,19 @@ module Serve = struct
     match Env_vars.perfetto_dir with
     | None -> Command.Param.return Disabled
     | Some perfetto_ui_base_directory ->
-      let%map_open.Command port =
+      let%map_open.Command serve =
+        flag "serve" no_arg ~doc:" Host the magic-trace UI locally."
+      and port =
         let default = 8080 in
         flag
-          "serve"
+          "port"
```

Comment:
What do you think about calling this `-serve-port` so it sorts next to `serve` in the help?

Reply:
I initially tried that, because ideally the sort order would be:

`-serve`
`-serve-port`

but `Command.t` sorted them in reverse order. Right now they sort

`-port`
`-serve`

so having them shorter seems nice, but I guess if we add any other flag that sorts in-between it'd be less nice...

I just switched back to `-serve-port`.